application/listeners/posthelpers.go

```diff
@@ -63,7 +63,73 @@ func sendNotificationRequestFromOpenToCommitted(template string, post m.Post) {
 	}
 }
 
-func sendNotificationRequestFromCommittedToAccepted(template string, post m.Post) {
+// Until we have status auditing history, we don't know who reverted the Post to `open` status.
```

Comment:
```suggestion
// Until we have status auditing history, we don't know who reverted the Post to `accepted` status.
```

Reply:
Heh, heh, I think the UI threw you off. You're suggestion appears to be based on the previous function name rather than the new one.